vyxal/elements.py

```diff
@@ -3097,6 +3105,18 @@ def run_length_decoding(lhs, ctx):
         return LazyList(temp)
 
 
+def sans_last_prepend_zero(lhs, ctx):
```

Comment:
Could we rename this `drop_last_prepend_zero` instead? `drop` feels more intuitive than `sans`, even if `sans` is fancier.

Reply:
^